src/services/local-storage.service.js

```diff
@@ -1,6 +1,6 @@
 export const clearLocalStorage = () => {
   const horondi = {
-    HORONDI_AUTH_TOKEN: null,
+    HORONDI_AUTH_TOKEN: '',
```

Comment:
Revert this change, it is better to have `null` as an empty value

Reply:
it helps to work with routes where we don`t use tokens. {token: null} works wrong